src/reducers/index.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+
+import bookReducer from './books';
+
+const rootReducre = combineReducers({
```

Comment:
[Optional]

- There is a typo. I think you wanted to write `rootReducer` instead of `rootReducre`

Reply:
Thanks, @Dipeshtwis for approving the project, and thanks also for your suggestions.